examples/tests/sdk_tests/bucket/delete.w

```diff
@@ -4,7 +4,18 @@ let b = new cloud.Bucket();
 b.addObject("file2.txt", "Bar");
 
 test "delete" {
-  let var error = "";
+  let assertThrows = (expected: str, block: (): void) => {
```

Comment:
Let's create a `testing` submodule in the Wing SDK and add it there?

```js
bring testing;

testing.assertThrows("message", () => { ... });
```

We can add `testing.assertEquals(x, y)` and `testing.assertNotEquals(x, y)` there as well to improve diagnostics for equality assertions.

@Chriscbr any objections?

Reply:
FYI: #3901

Reply:
@eladb Sounds good to me. I might suggest `assertEq` or `assertNotEq` just to save some verbosity

Reply:
FYI: #3918